src/applications/vaos/appointment-list/redux/selectors.js

```diff
@@ -644,21 +644,22 @@ export function selectConfirmedAppointmentData(state, appointment) {
   const locationId = getVAAppointmentLocationId(appointment);
 
   const { appointmentDetailsStatus, facilityData } = state.appointments;
-  const facility = facilityData?.[locationId];
+  const facility =
```

Comment:
Wow! Didn't know 'facilityData' was in 2 locations. We really need to cleanup.

Reply:
Didn't know either until I noticed appointment list was pulling from `appointment.vaos.facilityData` while the appointment detail page was pulling from `facilityData?.[locationId]`. This all revealed itself when testing the different null state ticket.